src/backend/joanie/payment/backends/lyra/__init__.py

```diff
@@ -213,6 +228,61 @@ def create_one_click_payment(self, order, billing_address, credit_card_token):
 
         return self._get_payment_info(url, payload)
 
+    def create_zero_click_payment(self, order, credit_card_token, amount):
+        """
+        Create a zero click payment object for a given order
+
+        https://docs.lyra.com/fr/rest/V4.0/api/kb/zero_click_payment.html
+        https://docs.lyra.com/fr/rest/V4.0/api/playground/Charge/CreatePayment
+        """
+
+        url = f"{self.api_url}Charge/CreatePayment"
+        payload = self._get_common_payload_data(order)
+        payload["amount"] = int(amount * 100)
+        payload["formAction"] = "SILENT"
+        payload["paymentMethodToken"] = credit_card_token
+
+        credit_card = CreditCard.objects.get(token=credit_card_token)
+        if (
+            initial_issuer_transaction_identifier
+            := credit_card.initial_issuer_transaction_identifier
+        ):
+            payload["transactionOptions"] = {
+                "cardOptions": {
+                    "initialIssuerTransactionIdentifier": initial_issuer_transaction_identifier
+                }
+            }
```

Comment:
If the credit card has no `initial_issuer_transaction_identifier` set, we should not raise a `ValueError` instead ? As from my understanding, it does not make sense to do a zero click payment with an null initialIssuerTransactionIdentifier.

Reply:
Oh ok, I believed `initial_issuer_transaction_identifier` was mandatory to do a zero click payment.